src/Raven.Server/Documents/BundleLoader.cs

```diff
@@ -60,7 +59,7 @@ public void HandleDatabaseRecordChange()
         /// </summary>
         public void InitializeBundles()
         {
-
+            PeriodicBackupRunner = new PeriodicBackupRunner(_database, _serverStore);
```

Comment:
There can be multiple backup configurations (on other bundles have configuration).
Updating one backup shouldn't reinitialize all the other backups